src/aleph/vm/orchestrator/views/__init__.py

```diff
@@ -318,3 +318,60 @@ async def update_allocations(request: web.Request):
         },
         status=status_code,
     )
+
+
+async def notify_allocation(request: web.Request):
+    if not authenticate_api_request(request):
+        return web.HTTPUnauthorized(text="Authentication token received is invalid")
+
+    try:
+        data = await request.json()
+        vm_notification = VMNotification.parse_obj(data)
+    except ValidationError as error:
+        return web.json_response(data=error.json(), status=web.HTTPBadRequest.status_code)
+
+    pubsub: PubSub = request.app["pubsub"]
+    pool: VmPool = request.app["vm_pool"]
+
+    instance = vm_notification.instance
+
+    # Exceptions that can be raised when starting a VM:
+    vm_creation_exceptions = (
+        UnknownHashError,
+        ResourceDownloadError,
+        FileTooLargeError,
+        VmSetupError,
+        MicroVMFailedInitError,
+        HostNotFoundError,
+    )
+
+    scheduling_errors: dict[ItemHash, Exception] = {}
+
+    instance_item_hash = ItemHash(instance)
```

Comment:
No, because the `VMNotification.instance` field is a `string`, so we need to convert to an `ItemHash`, exactly that it makes on the `update_allocation` method.